src/main/java/com/mraof/minestuck/client/gui/playerStats/InventoryEditmodeScreen.java

```diff
@@ -46,11 +48,19 @@ protected void renderBg(GuiGraphics guiGraphics, float par1, int xcor, int ycor)
 				|| m == Month.OCTOBER && d == 25 || m == Month.NOVEMBER && d == 11
 				|| m == Month.NOVEMBER && d == 27);
 		
-		guiGraphics.blit(GUI_BACKGROUND, xOffset+ LEFT_ARROW_X, yOffset+ ARROW_Y, guiWidth + (b2?36:0), (less?0:18) + (b1?36:0), 18, 18);
-		guiGraphics.blit(GUI_BACKGROUND, xOffset+ RIGHT_ARROW_X, yOffset+ ARROW_Y, guiWidth+18 + (b2?36:0), (more?0:18) + (b1?36:0), 18, 18);
+		guiGraphics.blit(GUI_BACKGROUND, xOffset + LEFT_ARROW_X, yOffset + ARROW_Y, guiWidth + (b2 ? 36 : 0), (less ? 0 : 18) + (b1 ? 36 : 0), 18, 18);
+		guiGraphics.blit(GUI_BACKGROUND, xOffset + RIGHT_ARROW_X, yOffset + ARROW_Y, guiWidth + 18 + (b2 ? 36 : 0), (more ? 0 : 18) + (b1 ? 36 : 0), 18, 18);
 		
 		drawActiveTabAndIcons(guiGraphics);
 		
+		//TODO add more options (potentially work with icons)
+		addRenderableWidget(new ExtendedButton(xOffset + LEFT_ARROW_X + 31, yOffset + ARROW_Y + 31, 100, 16, Component.literal("TELEPORT CLOSEST"), button -> teleport()));
```

Comment:
I like the suggestion, but im not sure I feel its more intuitive! I will agree that what I currently have put could use work. And yeah it needs a language key.

An alternative thought I have had would be to create a secondary menu of sorts where all editmode locations are listed. It could also have toggles for theoretical content such as enabling no-clip, disabling selection based block placing/breaking, moving individual blocks around without placing/breaking, or copy pasting selections of blocks

Reply:
I decided to go through with creating a pop up menu for additional editmode functionality. We could perhaps write out a section labelled "editmode sources" and have explanatory text when hovering over it? Open to suggestions!